src/app/public/modules/viewkeeper/viewkeeper-options.ts

```diff
@@ -0,0 +1,19 @@
+export interface SkyViewkeeperOptions {
+  fixToBottom?: boolean;
+
+  setWidth?: boolean;
+
+  id?: string;
+
+  el?: HTMLElement;
+
+  boundaryEl?: HTMLElement;
+
+  verticalOffset?: number;
+
+  isFixed?: boolean;
+
+  verticalOffsetEl?: HTMLElement;
+
+  viewportMarginTop?: number;
```

Comment:
Since this interface is exposed to the consumer, do you mind adding JSDoc comments above each property to explain their purpose? (These descriptions will be automatically used on the public-facing docs once we get docs-tools installed.)

Reply:
Adding the JSDoc comments gave me an opportunity to audit the properties and remove some unused ones, so good suggestion.